core/tests/ts-integration/package.json

```diff
@@ -1,36 +1,36 @@
 {
-    "name": "ts-integration",
-    "version": "0.1.0",
-    "license": "MIT",
-    "private": true,
-    "scripts": {
-        "test": "zk f jest --forceExit --testTimeout 60000",
-        "long-running-test": "zk f jest",
-        "fee-test": "RUN_FEE_TEST=1 zk f jest -- fees.test.ts",
-        "api-test": "zk f jest -- api/web3.test.ts api/debug.test.ts",
-        "contract-verification-test": "zk f jest -- api/contract-verification.test.ts",
-        "build": "hardhat compile",
-        "build-yul": "hardhat run scripts/compile-yul.ts"
-    },
-    "devDependencies": {
-        "@matterlabs/hardhat-zksync-deploy":  "^0.6.5",
-        "@matterlabs/hardhat-zksync-solc": "0.4.2",
-        "@matterlabs/hardhat-zksync-vyper": "^1.0.0",
-        "@nomiclabs/hardhat-vyper": "^3.0.5",
-        "@types/jest": "^29.0.3",
-        "@types/node": "^18.19.15",
-        "@types/node-fetch": "^2.5.7",
-        "chalk": "^4.0.0",
-        "ethereumjs-abi": "^0.6.8",
-        "ethers": "~5.7.0",
-        "hardhat": "=2.16.0",
-        "jest": "^29.0.3",
-        "jest-matcher-utils": "^29.0.3",
-        "node-fetch": "^2.6.1",
-        "ts-jest": "^29.0.1",
-        "ts-node": "^10.1.0",
-        "typescript": "^4.3.5",
-        "zksync-ethers": "https://github.com/zksync-sdk/zksync-ethers#sb-old-sdk",
-        "elliptic": "^6.5.5"
-    }
+  "name": "ts-integration",
+  "version": "0.1.0",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "test": "zk f jest --forceExit --testTimeout 120000",
```

Comment:
also, why did we change the padding for this file?

Reply:
I just changed the padding because it also changed in kl-factory, so just changing it back.

 the 120k is an error, will fix

Reply:
https://github.com/matter-labs/zksync-era/pull/298/files#r1572483926